### ISSUE TITLE
scrape and facebook online/offline status

### ISSUE THREAD

**Issue by totalmachine** [sqren/fb-sleep-stats#96]
Can scrape be put into silent mode, because when it runs I appear online on facebook?

**Comment by totalmachine** [comment]
I'd put my facebook chat on off, but friends I chat very often can see my timestamp if scrape runs at that time

**Comment by totalmachine** [comment]
So?

**Comment by sorenlouv** [comment]
No. The scraper works by logging in as you - therefore your status will be online.

**Comment by AmineIzumi** [comment]
Can't you set the offline status when logging in like in Messenger ("Appear offline") and Facebook web ("Disable chat") ? 
Even with the chat disabled, Facebook still shows timestamps :)

**Comment by totalmachine** [comment]
@AmineIzumi I've already mentioned that :)

**Comment by AmineIzumi** [comment]
Ah, just understood, I think : this doesn't use facebook website's chat but messenger website's chat, where you can't appear offline yet. My bad, sorry !

**Comment by totalmachine** [comment]
Even you appear offline, timestamp will be shown in both cases. So nevermind. You can turn off on facebook site or at messenger. It's same.
